demo/src/pages/index.js

```diff
@@ -147,6 +148,21 @@ const IndexPage = () => {
             `}
           />
           <br />
+          <h2>Two columns</h2>
+          <SideBySide>
+            <div>
```

Comment:
i'm curious why we are using `div` tags as part of this. ideally, we do not want to use `div` tags in MDX files. instead, can we use a child component like `Column`?

Reply:
We have to look into this once we do the `MDX` treatment, but also, the theme doesn't have the `Column` component. The goal is that writers don't have to deal with `div` tags at all.